main.js

```diff
@@ -0,0 +1,51 @@
+// 'use strict';
+
+{
+    
+    
+        function judge() {
+
+            const input_FizzNum = parseFloat(document.getElementById("FizzNum").value);//数値を取得
+            const input_BuzzNum = parseFloat(document.getElementById("BuzzNum").value);//数値を取得
+            
+            
+            if(Number.isInteger(input_FizzNum && input_BuzzNum) === true){
```

Comment:
以下のように入力値に不備があった場合は `return` で抜けるようにするとコードが見やすくなります。

```suggestion
            if (Number.isInteger(input_FizzNum && input_BuzzNum) === false) {
                const p = document.createElement('p');
                p.textContent = '整数値を入力してください';
                const body = document.querySelector('body');
                body.appendChild(p);
                return;                
            }
```